app/Models/Movie.php

```diff
@@ -3,11 +3,11 @@
 namespace App\Models;
 
 use Illuminate\Database\Eloquent\Model;
-use BeyondCode\Comments\Traits\HasComments;
+/*use BeyondCode\Comments\Traits\HasComments;*/
 
 class Movie extends Model
 {
-    use HasComments;
+    /*use HasComments;*/
```

Comment:
uncomment these and instead run "composer require beyondcode/laravel-comments --ignore-platform-reqs " inside your project root folder. The errors will be solved.